ansible/files/runtimes.json

```diff
@@ -58,7 +58,10 @@
                 "stemCells": [
                     {
                         "count": 2,
-                        "memory": "256 MB"
+                        "memory": "256 MB",
+                        "ttlMinutes": 10,
+                        "threshold": 5,
+                        "increment": 1
```

Comment:
In this scenario, I think "count" behaves like "initialCount" -right? so now I think we should also have a "maxCount" so that additional prewarms cannot be created past the maxCount limit.

Reply:
We could consider renaming "count" to "initialCount", but I'm not sure this is an issue for backwards compatibility.

Reply:
Already fixed